src/components/DetailsCard/CardMobile.tsx

```diff
@@ -15,6 +15,7 @@ const useStyles = makeStyles((theme) => ({
     width: '100%',
     borderRadius: 16,
     padding: theme.spacing(0),
+    boxShadow: '0 40px 50px 0 rgba(103, 118, 128, 0.1)',
```

Comment:
As this box-shadow is applied in many places, I would consider to move it to the theme

Reply:
I can see that box shadows are pretty random across the app, I will create an Issue to check them with designs and maybe add them to the theme there. Wdyt? 😄